data/gegno/ships.txt

```diff
@@ -0,0 +1,770 @@
+	sprite ship/schistmedb
+	thumbnail thumbnail/schistmedb
+	add attributes
+		mass 24
+		"cargo space" 24
+		drag 0.2
+
+ship Schist "Schist Heavy"
+	sprite ship/schistheavy
+	thumbnail thumbnail/schistheavy
+	add attributes
+		mass 30
+		"cargo space" 30
+		drag 0.4
+
+ship Eclogite
+	sprite ship/eclogite
+	thumbnail thumbnail/eclogite
+	swizzle 0
+	attributes
+		category "Light Warship"
+		cost 900000
+		mass 275
+		bunks 20
+		"cargo space" 70
+		drag 3.7
+		"engine capacity" 110
+		"fuel capacity" 400
+		"heat dissipation" 0.8
+		hull 6900
+		"outfit space" 375
+		"required crew" 8
+		shields 0
+		"weapon capacity" 90
+		weapon
+			"blast radius" 36
+			"shield damage" 210
+			"hull damage" 490
+			"hit force" 470
+	outfits
+		"C7 Brawl Core"
+		"Chloeric Cannon"
+		Hyperdrive
+		"Irate Cannon" 4
+		"R03 Warforge Battery"
+		"RG18 Torch Steering"
+		"RG18 Torch Thruster"
+	engine 0 70
+	gun 0 -94 "Chloeric Cannon"
+	gun -11 -57.5 "Irate Cannon"
+	gun 11 -57.5 "Irate Cannon"
+	gun -11 -38 "Irate Cannon"
+	gun 11 -38 "Irate Cannon"
+	leak leak 60 50
+	leak flame 50 80
+	explode "medium explosion" 10
+	explode "small explosion" 20
+	explode "tiny explosion" 15
+	"final explode" "final explosion small"
+	description "The Eclogite is a newer Gegno ship- a gunboat armed to the teeth with cannons, and maneuverability that the Gegno Vi's larger ships lack. It plays the role of a hit-and-run vessel, unloading volleys of shells against a target's armor in a pass before swinging around for another run."
+
+ship Hornfel
+	sprite ship/hornfel
+	thumbnail thumbnail/hornfel
+	swizzle 0
+	attributes
+		category "Medium Warship"
+		cost 1960000
+		mass 740
+		bunks 74
+		"cargo space" 120
+		drag 5.9
+		"engine capacity" 94
+		"fuel capacity" 400
+		"heat dissipation" 0.7
+		hull 14550
+		"outfit space" 428
+		"required crew" 58
+		shields 0
+		"weapon capacity" 140
+		weapon
+			"blast radius" 90
+			"shield damage" 600
+			"hull damage" 840
+			"hit force" 900
+	outfits
+		"C17 Warzone Core"
+		"Chloeric Turret" 2
+		Hyperdrive
+		"Irate Cannon" 4
+		"R03 Warforge Battery"
+		"RG18 Torch Steering"
+		"RG18 Torch Thruster"
+	engine -16.5 206
+	engine 16.5 206
+	gun -13.5 -228 "Irate Cannon"
+	gun 13.5 -228 "Irate Cannon"
+	gun -22.5 -204 "Irate Cannon"
+	gun 22.5 -204 "Irate Cannon"
+	turret 0 -112 "Chloeric Turret"
+	turret 0 -41.5 "Chloeric Turret"
+	leak leak 60 50
+	leak flame 60 80
+	explode "large explosion" 10
+	explode "medium explosion" 25
+	explode "small explosion" 25
+	explode "tiny explosion" 10
+	"final explode" "final explosion medium"
+	description "Hornfels are the powerhorse of the Gegno Vi's military. They're decently sized, decently fast, and armed more than decently to be able to fight on their own. Because the Gegno Vi are always requiring more replacement supplies in war, they double as a secondary cargo transport as well, and are seen occasionally with cargo pods attached to the several racks lining the sides of the ship."
```

Comment:
```suggestion
	description "Hornfels are the workhorse of the Gegno Vi's military. They're decently sized, decently fast, and more than decently armed in order to be able to fight on their own. Because the Gegno Vi always require more replacement supplies in war, they double as a secondary cargo transport as well, and are seen occasionally with cargo pods attached to the several racks lining the sides of the ship."
```